sdks/cpp/sdk.h

```diff
@@ -12,13 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include "sdk_global.h"
+
 #include <grpcpp/grpcpp.h>
 #include "sdk.grpc.pb.h"
 
 namespace agones {
 
     // The Agones SDK
-    class SDK {
+    class AGONES_EXPORT SDK {
```

Comment:
Out of curiosity - what does this do?

Reply:
This probably answers the question better than anything I could write:
https://stackoverflow.com/questions/8863193/what-does-declspecdllimport-really-mean

Basically, windows specific code for DLL usage.